sdcm/remote.py

```diff
@@ -43,6 +44,34 @@ class OutputCheckError(Exception):
     pass
 
 
+class CmdExecTimeoutExceeded(Exception):
```

Comment:
maybe we can inherit from `socket.timeout` ?

Reply:
Didn't think about. Will check how it will look like